tests/test-dirs/construct/c-arg-labels.t

```diff
@@ -0,0 +1,65 @@
+Check completion of values that are functions with various label types :
+
+  $ cat >c1.ml <<EOF
+  > let f_unlabeled_arg x = Some x
+  > let f_labeled_arg ~x = Some x
+  > let f_optional_arg ?(x=5) = Some x
+  > let f_position_arg ~(pos : [%call_pos]) x = Some x
+  > let x : int option = _
+  > EOF
+
+  $ $MERLIN single construct -with-values local -position 5:22 \
+  > -filename c1.ml <c1.ml | jq ".value"
+  [
+    {
+      "start": {
+        "line": 5,
+        "col": 21
+      },
+      "end": {
+        "line": 5,
+        "col": 22
+      }
+    },
+    [
+      "None",
+      "(Some _)",
+      "(f_unlabeled_arg _)",
+      "(f_labeled_arg ~x:_)",
+      "(f_optional_arg ?x:_)",
+      "(f_position_arg _)"
+    ]
+  ]
+
+Check completion of types that are functions with various label types :
```

Comment:
I'm happy with the output here (thanks for adding the test), but I don't see how this second test differs from the first.